extensions/digitalocean/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "digitalocean",
+  "title": "DigitalOcean",
+  "description": "Quickly find resources in your DigitalOcean team",
+  "icon": "command-icon.png",
+  "author": "shukantpal",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-container-registry",
+      "title": "Search Container Registry",
+      "subtitle": "DigitalOcean",
```

Comment:
This subtitle is actually redundant. If there is no title field, it already defaults to the extension title.

Reply:
That's only happening in development mode. I'm pretty sure that in production, it actually makes sense to have it like this.